validphys2/src/validphys/process_options.py

```diff
@@ -157,39 +155,48 @@ def _dijets_xq2map(kin_info):
 
 def _hqp_yq_xq2map(kin_info):
     # Compute x, Q2
-    if {"k1", "k2", "k3"} <= kin_info.keys():
-        kin_info[_Vars.y_t] = kin_info["k1"]
-        kin_info[_Vars.m_t2] = kin_info["k2"]
-        kin_info[_Vars.sqrts] = kin_info["k3"]
-
-    mass2 = kin_info.get_one_of(_Vars.m_t2, _Vars.m_ttBar)
-
+    # Theory predictions computed with HT/4 see 1906.06535
+    mass2 = kin_info[_Vars.m_t2]
     ratio = np.sqrt(mass2) / kin_info[_Vars.sqrts]
     x1 = ratio * np.exp(kin_info[_Vars.y_t])
     x2 = ratio * np.exp(-kin_info[_Vars.y_t])
     q2 = mass2
     x = np.concatenate((x1, x2))
-    return np.clip(x, a_min=None, a_max=1, out=x), np.concatenate((q2, q2))
+    return np.clip(x, a_min=None, a_max=1, out=x), np.concatenate((q2, q2)) / 4
 
 
 def _hqp_yqq_xq2map(kin_info):
     # Compute x, Q2
-    mass2 = kin_info.get_one_of(_Vars.m_t2, _Vars.m_ttBar)
+    # Theory predictions computed with HT/4 see 1906.06535
+    mass2 = kin_info[_Vars.m_t2]
     ratio = np.sqrt(mass2) / kin_info[_Vars.sqrts]
     x1 = ratio * np.exp(kin_info[_Vars.y_ttBar])
     x2 = ratio * np.exp(-kin_info[_Vars.y_ttBar])
     q2 = kin_info[_Vars.m_t2]
     x = np.concatenate((x1, x2))
-    return np.clip(x, a_min=None, a_max=1, out=x), np.concatenate((q2, q2))
+    return np.clip(x, a_min=None, a_max=1, out=x), np.concatenate((q2, q2)) / 4
 
 
 def _hqp_ptq_xq2map(kin_info):
     # Compute x, Q2
-    QMASS2 = TMASS * TMASS
-    Q = np.sqrt(QMASS2 + kin_info[_Vars.pT_t] * kin_info[_Vars.pT_t]) + kin_info[_Vars.pT_t]
+    Q = (kin_info[_Vars.m_t2] + kin_info[_Vars.pT_t] * kin_info[_Vars.pT_t]) ** 0.5 / 2
```

Comment:
Why divided by 2? Wasn't it divided by 4?

Reply:
The paper gives a different expression for pT

Reply:
I thought you were always using HT/4?

Reply:
> I thought you were always using HT/4?

true dat! but I could do that in MATRIX, I can't do that here as I don't have the pT of tbar

Reply:
for ref, this is HT/4: 
<img width="324" alt="image" src="https://github.com/NNPDF/nnpdf/assets/99052355/d69a5247-48fc-4d6d-b9a4-b77c5f9a5bcd">

Reply:
At LO pT_t == pT_tb since they are back-to-back.

But I see, HT/4 ~ sqrt(mt^2 + pt^2) / 2

Could you also write the expression for HT in one of the functions as well please?